client/src/CommentListItem.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
 const CommentListItem = (props) => (
```

Comment:
this component can just be called Comment

Reply:
Updated CommentListItem to just Comment